content/influxdb/cloud/organizations/buckets/bucket-schema/_index.md

```diff
@@ -97,6 +97,18 @@ Use the `influx` CLI to set the schema-type and measurement schemas for your buc
       ]
       ```
 
+   The default field data type is `string`.
+   To set the data type of a field column, provide the `dataType` property and a valid
+   [field data type](/influxdb/cloud/reference/key-concepts/data-elements/#field-value),
```

Comment:
thoughts on:
- explicitly stating here "strings, floats, integers, or booleans" so folks don't have to click
- and/or rephrasing valid data type for the [field value]

Reply:
also looks like we may need to add "timestamp" to the list in the "Field value" section?
per https://docs.influxdata.com/influxdb/cloud/reference/glossary/#data-type
also, would it make sense to clarify the distinction between integer and unsigned integer?